pycbc/events/stat.py

```diff
@@ -2435,7 +2436,10 @@ def get_statistic_from_opts(opts, ifos):
         opts.statistic_keywords = []
 
     # flatten the list of lists of filenames to a single list (may be empty)
-    opts.statistic_files = sum(opts.statistic_files, [])
+    # if needed (e.g. not calling get_statistic_from_opts in a loop)
```

Comment:
As this edits the input argparser namespace, we need to avoid redoing this step multiple times